Brokkr/src/main/include/commands/ArmPosition.h

```diff
@@ -30,5 +32,6 @@ class ArmPosition
 
   private:
   Arm& mArm;
+  frc2::PIDController mArmControl{0, 0, 0};
```

Comment:
This is yet another way of doing controllers in commands. 

Once we can do testing and verify operations, perhaps we can decide on which method we want to use.

No change needed.